docs/source/recipes.rst

```diff
@@ -224,89 +188,89 @@ You can also use *iterators* (including *generators*) to provide multiple values
 
     from itertools import cycle
 
-    colors = ['red', 'green', 'blue', 'yellow']
-    person = Recipe(Person,
-        favorite_color = cycle(colors)
+    names = ['Ada Lovelace', 'Grace Hopper', 'Ida Rhodes', 'Barbara Liskov']
+    customer = Recipe(Customer,
+        name=cycle(names)
     )
 
-*Mommy* will use the next value in the *iterator* every time you create a model from the recipe.
+Model Bakery will use the next value in the *iterator* every time you create a model from the recipe.
 
 Sequences in recipes
 --------------------
 
-Sometimes, you have a field with an unique value and using `make` can cause random errors. Also, passing an attribute value just to avoid uniqueness validation problems can be tedious. To solve this you can define a sequence with `seq`
+Sometimes, you have a field with an unique value and using ``make`` can cause random errors. Also, passing an attribute value just to avoid uniqueness validation problems can be tedious. To solve this you can define a sequence with ``seq``
 
 .. code-block:: python
 
 
-    from model_mommy.recipe import Recipe, seq
-    from family.models import Person
+    from model_bakery.recipe import Recipe, seq
+    from shop.models import Customer
 
-    person = Recipe(Person,
-        name = seq('Joe'),
-        age = seq(15)
+    customer = Recipe(Customer,
+        name=seq('Joe'),
+        age=seq(15)
     )
 
-    p = mommy.make_recipe('myapp.person')
+    p = baker.make_recipe('shop.customer')
```

Comment:
```suggestion
    new_customer = baker.make_recipe('shop.customer')
```
This is only a suggestion. I just wanted to avoid one letter variables :sweat_smile:

Reply:
Fixed by 6fdba98